reactivate types checking

MonPote opened the pull request:
**Component**:

<!-- E.g. 'salt', 'containers', 'kubernetes', 'build', 'tests'... -->

**Context**: 

**Summary**:

**Acceptance criteria**: 


---

<!-- Declare one or more issues to close once this PR gets merged -->

Closes: #ISSUE_NUMBER

<!-- If you want to refer to an issue while not closing it, use:

See: #ISSUE_NUMBER

-->

MonPote committed:
fix

bert-e commented:
# Hello monpote,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://bert-e.scality.net/scality/metalk8s/doc/user).


<details>
  <summary><b>Available options</b></summary>

  name   | description  | privileged   | authored
  ------ | ------------ | ------------ |----------
  `/after_pull_request` | Wait for the given pull request id to be merged before continuing with the current one. |  | 
  `/bypass_author_approval` | Bypass the pull request author's approval |  :star:  | 
  `/bypass_build_status` | Bypass the build and test status |  :star:  | 
  `/bypass_commit_size` | Bypass the check on the size of the changeset ```TBA``` |  :star:  | 
  `/bypass_incompatible_branch` | Bypass the check on the source branch prefix |  :star:  | 
  `/bypass_jira_check` | Bypass the Jira issue check |  :star:  | 
  `/bypass_peer_approval` | Bypass the pull request peers' approval |  :star:  | 
  `/bypass_leader_approval` | Bypass the pull request leaders' approval |  :star:  | 
  `/approve` | Instruct Bert-E that the author has approved the pull request. |  |  :writing_hand: 
  `/create_pull_requests` | Allow the creation of integration pull requests. |  | 
  `/create_integration_branches` | Allow the creation of integration branches. |  | 
  `/no_octopus` | Prevent Wall-E from doing any octopus merge and use multiple consecutive merge instead |  | 
  `/unanimity` | Change review acceptance criteria from `one reviewer at least` to `all reviewers` |  | 
  `/wait` | Instruct Bert-E not to run until further notice. |  | 
  

</details>



<details>
  <summary><b>Available commands</b></summary>

  name   | description  | privileged
  ------ | ------------ | ------------
  `/help` | Print Bert-E's manual in the pull request. | 
  `/status` | Print Bert-E's current status in the pull request ```TBA``` | 
  `/clear` | Remove all comments from Bert-E from the history ```TBA``` | 
  `/retry` | Re-start a fresh build ```TBA``` | 
  `/build` | Re-start a fresh build ```TBA``` | 
  `/force_reset` | Delete integration branches & pull requests, and restart merge process from the beginning. | 
  `/reset` | Try to remove integration branches unless there are commits on them which do not appear on the source branch. | 
  
</details>




*Status report is not available.*

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @xaf-scality

* @jbwatenbergscality

* @m4nch0t

* @teddyandrieux

* @thomasdanan

* @eg-ayoub

* @aprucolimartins

* @ezekiel-alexrod

* @chengyanjin

MonPote committed:
fix types

MonPote committed:
fix shell-ui types

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @jbwatenbergscality

* @m4nch0t

* @teddyandrieux

* @ezekiel-alexrod

* @aprucolimartins

* @thomasdanan

* @xaf-scality

* @eg-ayoub

* @chengyanjin

MonPote committed:
update @mf-types for metalk8s-ui

MonPote committed:
update core-ui and module federation

MonPote committed:
reactivate type checking

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @teddyandrieux

* @eg-ayoub

* @aprucolimartins

* @xaf-scality

* @thomasdanan

* @chengyanjin

* @ezekiel-alexrod

* @jbwatenbergscality

* @m4nch0t

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @teddyandrieux

* @jbwatenbergscality

* @ezekiel-alexrod

* @xaf-scality

* @aprucolimartins

* @m4nch0t

* @thomasdanan

* @chengyanjin

* @eg-ayoub

MonPote committed:
recreate package-lock.json

MonPote committed:
align react @type version to others repositories

add @mf-types to .gitignore

remove unused type definitions and package.json files from react-query and core-ui

Revert "remove unused type definitions and package.json files from react-query and core-ui"

This reverts commit 0352388e98a79350f69a1a5d60280c2aefc5fb63.

Revert "add @mf-types to .gitignore"

This reverts commit 84dff9182a0bbbeb123c3f9fe4d58c0fab714e26.

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @teddyandrieux

* @aprucolimartins

* @chengyanjin

* @m4nch0t

* @thomasdanan

* @jbwatenbergscality

* @ezekiel-alexrod

* @xaf-scality

* @eg-ayoub

MonPote commented:
/approve

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @teddyandrieux

* @jbwatenbergscality

* @eg-ayoub

* @aprucolimartins

* @chengyanjin

* @ezekiel-alexrod

* @xaf-scality

* @m4nch0t

* @thomasdanan










*The following options are set:* **approve**

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @ezekiel-alexrod

* @teddyandrieux

* @m4nch0t

* @chengyanjin

* @eg-ayoub

* @xaf-scality

* @aprucolimartins

* @thomasdanan

* @jbwatenbergscality










*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/129.0`



The following branches have **NOT** changed:

* `development/123.0`
* `development/124.0`
* `development/124.1`
* `development/125.0`
* `development/126.0`
* `development/127.0`
* `development/128.0`
* `development/2.0`
* `development/2.1`
* `development/2.10`
* `development/2.11`
* `development/2.2`
* `development/2.3`
* `development/2.4`
* `development/2.5`
* `development/2.6`
* `development/2.7`
* `development/2.8`
* `development/2.9`



Please check the status of the associated issue ARTESCA-13131.

Goodbye monpote.




*The following options are set:* **approve**
